livekit-plugins/livekit-plugins-azure/livekit/plugins/azure/stt.py

```diff
@@ -330,6 +333,11 @@ def _create_speech_recognizer(
             speechsdk.enums.PropertyId.Speech_SegmentationStrategy,
             str(config.segmentation_strategy),
         )
+    if config.profanity_option:
+        speech_config.set_property(
+            speechsdk.enums.PropertyId.SpeechServiceResponse_ProfanityOption,
+            str(config.profanity_option.name.lower()),
```

Comment:
according to the [docs](https://learn.microsoft.com/en-us/azure/ai-services/speech-service/display-text-format?pivots=programming-language-python), this can just be:

```
speech_config.set_profanity(config.profanity)
```